test/e2e/minor_version_compatibility.go

```diff
@@ -55,7 +55,9 @@ func MinorVersionCompatibility(logger *log.Logger) error {
 		// each node begins with a random version within the same major version set
 		v := versions.Random(r).String()
 		logger.Println("Starting node", "node", i, "version", v)
-		testnet.NoError("failed to create genesis node", testNet.CreateGenesisNode(v, 10000000, 0, testnet.DefaultResources))
+		testnet.NoError("failed to create genesis node",
+			testNet.CreateGenesisNode(v, 10000000, 0,
+				testnet.DefaultResources, false))
```

Comment:
To elaborate on my previous comment: my perspective is that the bbr flag must be set according to the version of the celestia-app used for the validators. Since some versions support this flag while others do not, it is being made a parameter in the `CreateGenesisNode` function. This allows the caller, based on the version specified (the first parameter), to determine whether to enable or disable bbr.